meson.build

```diff
@@ -232,6 +232,10 @@ subdir('pldmtool')
 
 subdir('configurations')
 
+if get_option('oem-ibm').enabled()
+  subdir('oem/ibm/pfw-sms-utils')
```

Comment:
can we create this folder under oem/ibm/libpldmresponder ?

Reply:
thinking again , i think we are okay with oem/ibm/pfw-sms-utils , as its a util API and does not really not a util libpldmresponder API. please ignore my comment about movement.